py3status/modules/wwan.py

```diff
@@ -535,18 +538,19 @@ def wwan(self):
 
         # notifications
         if self.format_notification:
-            composite = self.py3.safe_format(self.format_notification,
-                                             wwan_data)
+            composite = self.py3.safe_format(self.format_notification, wwan_data)
             notification = self.py3.get_composite_string(composite)
 
-            if notification != self.last_notification:
+            if notification != '' and notification != self.last_notification:
```

Comment:
I'd try undo this... and try the `None` above. The `self.py3.notify_user()` code will skip if the notification is empty... so this might not be needed. That's all.